src/main/java/baubles/common/event/EventHandlerNetwork.java

```diff
@@ -23,9 +23,13 @@ public void playerLoggedInEvent(PlayerEvent.PlayerLoggedInEvent event)    {
 		if (side == Side.SERVER)        {
 			// Apply all baubles
             InventoryBaubles baubles = PlayerHandler.getPlayerBaubles(event.player);
+            PacketHandler.INSTANCE.sendToAll(new PacketSyncAllBauble(event.player));
```

Comment:
This line is superfluous, since the current player is already part of the `playerEntityList` next line. It sends the packet twice, leading to baubles being applies twice (like reach ring)

Reply:
sigh.. I was meant to exempt the current player from full sync but I forgot. should add it back